bitcrowd/bitstyles
# [#750] Update Flash component

Fixes #750 

## Changes

-  Adds a simple Flash component for use in stories
- Moves all the stories to flash.stories.js
- Flash colors handled by themes, only padding is specified in flash/settings
- Adds a very simple Icon component and uses it new 'with-icon' flash stories
- Adds `content` design tokens to share the horizontal spacing/padding of content between components, so flash content aligns to content.

## 📸

<img width="1028" alt="Screenshot 2023-04-03 at 12 22 19" src="https://user-images.githubusercontent.com/2479422/229482803-333f2008-e40d-4540-862e-b040b2242607.png">

## How to QA Visual changes

- `git checkout main` 
- `git pull` 
- `git checkout feature/750-update-flash-component` 
- `yarn`
- `yarn storybook`
- navigate to Flash component

Check:

- [ ] The appearance matches the zeplin designs

## How to QA code changes


## Preflight checks

PR author to check, and delete if not applicable:
